### DISCUSSION TITLE
Download loads every file of stacked photos

### DISCUSSION THREAD

**Discussion by mills1975** [photoprism/photoprism#4486] [Q&A]
Hello,

I've got a problem with downloading pics from photoprism.
When I'm downloading a picture which has stacked files, it downloads every file.
I'd expect, that the download would only bring the primary file of the stack, or am I missing something?

Many greetings
Andi

**Comment by graciousgrey** [comment]
Hello :wave: 

This depends on your download settings and the kind of stacks.

https://docs.photoprism.app/user-guide/settings/general/#downloads

You can e.g. disable downloading raw files or files from the sidecar folder (such as JPGs that have been generated by PhotoPrism).

More advanced download options that let you also choose the exact file and size of the file  are planned for the future.

**Reply by mills1975** [reply]
Hi,
thanks for your reply.

My problem isn't related to raw-files. (I've already disabled this download option)
In my case, sometimes I have no Raw's, but only jpegs as originals.
And so my Photoeditor makes a "xxxxx-1.jpg" when editing the pic.
In the webview, these picture are stacked and only shown once, as it should be.
But the problem is, when I downloading the pictures, I get the "xxxxx.jpg" and the "xxxxx-1.jpg" as well.

Many greetings
Andi

**Reply by graciousgrey** [reply]
There is currently no setting to download only the primary file of a stack. Such a setting would need to be carefully thought through as it does not make sense for all types of stacks (e.g. videos or live photos). We will keep this in mind the next time we improve the download settings.

When downloading files from the files tab of the edit dialog you can choose which file to download:
https://docs.photoprism.app/user-guide/organize/edit/#files
